audio-worklet/lib/param-controller.js

```diff
@@ -35,7 +35,17 @@ class ParamController {
     this.id_ = options.id || this.name_;
     this.controller_.type = options.type || 'range';
     this.controller_.min = options.min || 0;
-    this.controller_.max = options.max || 0;
+
+    if (options.max == null)
+      this.controller_.max = 1;
+    else
+      this.controller_.max = options.max;
```

Comment:
Passing `null` as argument is very uncommon practice. I think what you meant was `undefined` when you pass nothing. JS does coercion and no type checking, so there is only so much you can do.